cms/djangoapps/contentstore/views/import_export.py

```diff
@@ -158,6 +158,11 @@ def _write_chunk(request, courselike_key):
             mode = "wb+"
         else:
             mode = "ab+"
+            if not temp_filepath.exists():
+                _save_request_status(request, courselike_string, -1)
+                log.error(f'Course Import: {courselike_key} Chunks missed during upload.')
```

Comment:
I think is the wrong place to be logging the chunk missing error? I already added the missing chunk log in line 
#167

Reply:
good point, 
`size = os.path.getsize(temp_filepath)` would fail / break if a chunk is sent to a new server where file currently doesn't exist. 
Observe the scenario. 
1. Chunks (C1 & C2) sent to Server (S1).
2. Chunk (C3) is sent to server S2 where the file doesn't exist, so while checking the size of the file, the code will break.

Reply:
line #167 is for the case where.

* Chunk (C4) is received on Server 1 where size mismatches.